### PR TITLE
show the language of the ingredients displayed, fixes #3620

### LINKED ISSUE #3620: Show in which language the ingredient list is when it is not in the language of the interface
**Describe the bug**
On the main product page the ingredients are noted as english, but they are danish.

**To Reproduce**
1. Go to https://world.openfoodfacts.org/product/42115670/fanta-free-orange
2. Try to edit
**Expected behavior**

**Screenshots**
Screenshot 2020-06-13 at 13 50 53

### PR DESCRIPTION
**Pull request by stephanegigandet**
Fixes #3620 reported by @aleene 

I'll also add a prompt like "If this product has a list of ingredients in [language of the interface], you can add it by editing the product page".

### CODE REVIEW

File: lib/ProductOpener/Display.pm [hunk 1]
```diff
@@ -7749,15 +7755,14 @@ HTML
...
 
-<div class="button_div" id="editingredientsbuttondiv"><button id="editingredients" class="small button" type="button">Edit ingredients ($ilc)</div>
+<div class="button_div" id="editingredientsbuttondiv"><button id="editingredients" class="small button" type="button">Edit ingredients$ingredients_text_lang_html</div>
 <div class="button_div" id="saveingredientsbuttondiv_status" style="display:none"></div>
-<div class="button_div" id="saveingredientsbuttondiv" style="display:none"><button id="saveingredients" class="small button" type="button">Save ingredients ($ilc)</div>
+<div class="button_div" id="saveingredientsbuttondiv" style="display:none"><button id="saveingredients" class="small button" type="button">Save ingredients$ingredients_text_lang_html</div>
 
 
-<div class="button_div" id="wipeingredientsbuttondiv"><button id="wipeingredients" class="small button" type="button">Ingredients ($ilc) are completely bogus, erase them.</button></div>
+<div class="button_div" id="wipeingredientsbuttondiv"><button id="wipeingredients" class="small button" type="button">Ingredients$ingredients_text_lang_html are completely bogus, erase them.</button></div>
```
Comment (teolemon) [inline lib/ProductOpener/Display.pm:7765]:
```suggestion
<div class="button_div" id="wipeingredientsbuttondiv"><button id="wipeingredients" class="small button" type="button">Ingredients $ingredients_text_lang_html are completely bogus, erase them.</button></div>
```

File: lib/ProductOpener/Display.pm [hunk 2]
```diff
@@ -7749,15 +7755,14 @@ HTML
...
 			my $ilc = $ingredients_text_lang;
 
-
 			$html .= <<HTML
 
-<div class="button_div" id="editingredientsbuttondiv"><button id="editingredients" class="small button" type="button">Edit ingredients ($ilc)</div>
+<div class="button_div" id="editingredientsbuttondiv"><button id="editingredients" class="small button" type="button">Edit ingredients$ingredients_text_lang_html</div>
 <div class="button_div" id="saveingredientsbuttondiv_status" style="display:none"></div>
-<div class="button_div" id="saveingredientsbuttondiv" style="display:none"><button id="saveingredients" class="small button" type="button">Save ingredients ($ilc)</div>
+<div class="button_div" id="saveingredientsbuttondiv" style="display:none"><button id="saveingredients" class="small button" type="button">Save ingredients$ingredients_text_lang_html</div>
```
Comment (teolemon) [inline lib/ProductOpener/Display.pm:7762]:
```suggestion
<div class="button_div" id="saveingredientsbuttondiv" style="display:none"><button id="saveingredients" class="small button" type="button">Save ingredients $ingredients_text_lang_html</div>
```

File: lib/ProductOpener/Display.pm [hunk 3]
```diff
@@ -7749,15 +7755,14 @@ HTML
 
 			my $ilc = $ingredients_text_lang;
 
-
 			$html .= <<HTML
 
-<div class="button_div" id="editingredientsbuttondiv"><button id="editingredients" class="small button" type="button">Edit ingredients ($ilc)</div>
+<div class="button_div" id="editingredientsbuttondiv"><button id="editingredients" class="small button" type="button">Edit ingredients$ingredients_text_lang_html</div>
```
Comment (teolemon) [inline lib/ProductOpener/Display.pm:7760]:
```suggestion
<div class="button_div" id="editingredientsbuttondiv"><button id="editingredients" class="small button" type="button">Edit ingredients $ingredients_text_lang_html</div>
```

Comment (hangy) [reply lib/ProductOpener/Display.pm:7760]: `$ingredients_text_lang_html` already contains a space

Comment (CharlesNepote) [reply lib/ProductOpener/Display.pm]: Just noticed that these changes have broke Power User Script that needed to read `($ilc)` to call Google Cloud Vision. Of course "Spanish" is better for humans. But could you add a `data-lc="es"` in the `<div id="editingredients"`>?

Sorry to see that so late...

**Pull request merged**